.travis.yml

```diff
@@ -29,6 +34,10 @@ matrix:
       # quic-trace-log
       - RUSTFLAGS="-D warnings" cargo build --release --verbose --manifest-path tools/quic-trace-log/Cargo.toml
       - cargo clippy --manifest-path tools/quic-trace-log/Cargo.toml -- -D warnings
+   - name: "Docker images"
```

Comment:
If I understand this correctly this is going to cause the images being published every time someone pushes to any branch or PR. You need to use the deploy section for this so the images are only published when things get merged to master, see https://github.com/cloudflare/quiche/blob/master/.travis.yml#L133

Reply:
That is, we should still do `build` here though, to make sure it keeps working, but `publish` should go in the deploy section.

Reply:
I moved `docker push` to `deploy` section.